enterprise/models.py

```diff
@@ -2235,14 +2237,27 @@ class Meta:
         'placeholder {course_name} will be replaced with the name of the course or program that was enrolled in.'
     )
 
+    DEFAULT_TEMPLATE_TYPE = 'DE'
+    BULK_ENROLL_TEMPLATE_TYPE = 'BE'
+    template_type_choices = [
+        (DEFAULT_TEMPLATE_TYPE, 'Default Enrollment Template'),
+        (BULK_ENROLL_TEMPLATE_TYPE, 'Bulk Enrollment Template'),
+    ]
+
     plaintext_template = models.TextField(blank=True, help_text=BODY_HELP_TEXT)
     html_template = models.TextField(blank=True, help_text=BODY_HELP_TEXT)
     subject_line = models.CharField(max_length=100, blank=True, help_text=SUBJECT_HELP_TEXT)
-    enterprise_customer = models.OneToOneField(
+    enterprise_customer = models.ForeignKey(
         EnterpriseCustomer,
-        related_name="enterprise_enrollment_template",
+        related_name="enterprise_enrollment_templates",
```

Comment:
This can be left on what it was originally

Reply:
hmm, so if we store two different templates for a given customer, with different types, can we still use onetoone with a nullable value, or do we need the foreign key/ manyToOne method?

Reply:
Great point! If we see the possibility of like customer customized bulk enrollment templates, then yeah it should actually be a many to one (the many being templates, the one being customer)

Reply:
The current structure seems to be default, bulk, *or* customer, so in this structure customer would be a type and there wouldn't be a many to one. If we do see having a type of bulk with a customer specification then we need to figure out what type our existing customer templates are - default? Makes the use of "default" a bit odd no?

Reply:
yup exactly, we are setting up the bulk enroll templates the same way as the other templates, default in files, then per customer override (if desired) in the table